coderd/rbac/USAGE.md

```diff
@@ -0,0 +1,344 @@
+| **orgAdmin**         | Like **owner**, but scoped to a single organization                 | _(org-level equivalent)_                     |
+| **orgMember**        | Like **member**, but scoped to a single organization                | _(org-level equivalent)_                     |
+| **orgAuditor**       | Like **auditor**, but scoped to a single organization               | _(org-level equivalent)_                     |
+| **orgUserAdmin**     | Like **userAdmin**, but scoped to a single organization             | _(org-level equivalent)_                     |
+| **orgTemplateAdmin** | Like **templateAdmin**, but scoped to a single organization         | _(org-level equivalent)_                     |
+
+_\* except some, which are not important to this overview_
+
+## Actions
+
+Roles are collections of permissions (we call them _actions_).
+
+These can be found in `coderd/rbac/policy/policy.go`.
+
+| Action                  | Description                             |
+| ----------------------- | --------------------------------------- |
+| **create**              | Create a resource                       |
+| **read**                | Read a resource                         |
+| **update**              | Update a resource                       |
+| **delete**              | Delete a resource                       |
+| **use**                 | Use a resource                          |
+| **read_personal**       | Read owned resource                     |
+| **update_personal**     | Update owned resource                   |
+| **ssh**                 | SSH into a workspace                    |
+| **application_connect** | Connect to workspace apps via a browser |
+| **view_insights**       | View deployment insights                |
+| **start**               | Start a workspace                       |
+| **stop**                | Stop a workspace                        |
+| **assign**              | Assign user to role / org               |
+
+# Creating a new noun
+
+In the following example, we're going to create a new RBAC noun for a new entity
+called a "frobulator" _(just some nonsense word for demonstration purposes)_.
+
+_Refer to https://github.com/coder/coder/pull/14055 to see a full
+implementation._
+
+## Creating a new entity
+
+If you're creating a new resource which has to be owned by users of differing
+roles, you need to create a new RBAC resource.
+
+Let's say we're adding a new table called `frobulators` (we'll use this table
+later):
+
+```sql
+CREATE TABLE frobulators
+(
+  id           uuid NOT NULL,
+  user_id      uuid NOT NULL,
+  model_number TEXT NOT NULL,
+  PRIMARY KEY (id),
+  UNIQUE (model_number),
+  FOREIGN KEY (user_id) REFERENCES users (id) ON DELETE CASCADE
+);
+```
+
+Let's now add our frobulator noun to `coderd/rbac/policy/policy.go`:
+
+```go
+    ...
+	"frobulator": {
+		Actions: map[Action]ActionDefinition{
+			ActionCreate: actDef("create a frobulator"),
```

Comment:
Just to document a reason for the `actDef()` function. We can move to 
```
ActionCreate:: {Description: "create a frobulator"}
```

Originally there were more fields, and the helper function kept things as a single line. I removed the extra fields that were attempting to apply even more structure and rules on when to use a verb.

Now the function `actDef` is effectively pointless.